networking/multiple_networks/configuring-multi-network-policy.adoc

```diff
@@ -10,8 +10,10 @@ As a cluster administrator, you can configure network policy for additional netw
 
 [NOTE]
 ====
-You can specify multi-network policy for only macvlan additional networks.
-Other types of additional networks, such as ipvlan, are not supported.
+You can specify multi-network policy for SR-IOV and macvlan additional networks.
+Other types of additional networks, such as ipvlan, are not supported. 
+SR-IOV additional networks are technology preview and macvlan additional networks are fully supported. 
```

Comment:
```suggestion
SR-IOV additional networks are a Technology Preview feature, and macvlan additional networks are fully supported. 
```
"SR-IOV additonal networks" is a little bit awkward to my ear. Is this the standard term for this feature? WDYT of "SR-IOV support for additonal networks"? It isn't as minimal, but I think flows a little better.

Reply:
@michaelryanpeter I believe that changes the understanding SR-IOV is the additional network type so is macvlan. These are the two additional network types supported.